pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Sure, this would be a lot better, and is very much a step in the right direction. What I've done so far is to just ignore the Elasticsearch package in unattended-upgrades.

I'm more thinking in going forward. Maybe we don't want to help unattended upgrades at all, since it would break the ES node in most scenarios where a plugin is installed :)